### ISSUE TITLE
 annyang.trigger

### ISSUE THREAD

**Issue by lawrencenull** [TalAter/annyang#357]
## Expected Behavior
The  annyang.trigger event should pass a command and be able to accept tags, etc even (especially) when  annyang is not listening. When speech isn't available an input field is normally shown, and it would be nice to have a trigger event that would parse it the exact same as a speech command. This would provide a great deal of continuity to the end user.

## Possible Solution
 annyang.trigger('go to *tag') function(tag) {
			var str = tag;
			str = str.replace(/\s+/g, "");
			window.open("https://" + str);
		},

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Browser Name and version:
* Operating System and version (desktop or mobile):
* Link to your project:

**Comment by nerkdesign** [comment]
Hi, did you try this solution ? I'm interesting to use annyang using a input text and your solution if it works, can be a clue to do that. thanks

**Comment by lawrencenull** [comment]
Nope, what I ended up doing was parse the input value separately.

**Comment by TalAter** [comment]
This is now fully supported in **annyang v3.0.0** (just released).

`trigger()` works independently of speech recognition — you can call it whether annyang is listening, paused, or even in browsers that don't support speech recognition at all. This makes it perfect for fallback input fields.

```js
import annyang from 'annyang';

annyang.addCommands({
  'search for *term': (term) => { console.log(`Searching for ${term}`); },
});

if (annyang.isSpeechRecognitionSupported()) {
  annyang.start();
} else {
  // trigger() works even without speech recognition —
  // great for text input fallback
  annyang.trigger('search for cat videos');
}
```
